x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table.tsx

```diff
@@ -78,56 +86,75 @@ export const SpikeAnalysisTable: FC<SpikeAnalysisTableProps> = ({
     },
     {
       field: 'pValue',
-      name: 'p-value',
+      name: (
+        <EuiToolTip position="top" content={i18n.translate(
+          'xpack.aiops.correlations.failedTransactions.correlationsTable.pValueColumnTooltip',
+          { defaultMessage: 'For statistically significant changes in the distribution of values, indicates how extreme the change is; lower values indicate greater change.' }
```

Comment:
```suggestion
          { defaultMessage: 'For statistically significant changes in the frequency of values, indicates how extreme the change is; lower values indicate greater change.' }
```

Reply:
Thanks! Updated all in d5825358ea411ef8efd55cfecbce57184d1a3af2